.github/ISSUE_TEMPLATE/2-feature_request.yml

```diff
@@ -0,0 +1,37 @@
+name: Feature request 🚀
+description: Existing feature does not behave as expected.
```

Comment:
Same description than bug_report

Reply:
Good catch! Yeah, I copy-pasted a template from a bug report and it seems like I missed it 😅.

```suggestion
description: I have a suggestion about a new feature (and may want to implement it)!
```